packages/local_auth/lib/local_auth.dart

```diff
@@ -92,27 +92,30 @@ class LocalAuthentication {
               'operating systems.',
           details: 'Your operating system is ${_platform.operatingSystem}');
     }
-    return await _channel.invokeMethod<bool>(
-        'authenticateWithBiometrics', args);
+    final bool? result =
+        await _channel.invokeMethod<bool>('authenticateWithBiometrics', args);
```

Comment:
this could be `bool` if `(await _channel.invokeMethod<bool>('authenticateWithBiometrics', args))!`

Reply:
Yes. I find that less readable though. In the end, I return a `bool`.

Reply:
I see your point on readability. Feel free to ignore my comment. I think the real issue is having an `invokeMethod` method signature that ensures non-nullable in the native side as well as the Dart side. 

cc @gaaclarke looks like Pigeon could do that if the type is defined as non-null.